### PR TITLE
beachfront: Changes to support real 204

### PR DESCRIPTION
**Pull request by RodionOrets**
`go` repo PR: https://github.com/prebid/prebid-server/pull/1737/

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java [hunk 1]
```diff
@@ -203,6 +203,8 @@ private BeachfrontBannerRequest getBannerRequest(BidRequest bidRequest, List<Imp
             requestBuilder.secure(firstImpSecure != null ? firstImpSecure : getSecure(bundle));
         }
 
+        requestBuilder.real204(true);
```
Comment (snahornyi) [inline src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java:206]: Can we move this to 173 line?

File: src/main/java/org/prebid/server/bidder/beachfront/model/BeachfrontBannerRequest.java [hunk 2]
```diff
@@ -48,4 +48,6 @@
 
     @JsonProperty("requestId")
     String requestId;
+
+    boolean real204;
```
Comment (snahornyi) [inline src/main/java/org/prebid/server/bidder/beachfront/model/BeachfrontBannerRequest.java:52]: we prefer to use wrapped types

**Review by snahornyi — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java (hunk 1 above)
Comment (RodionOrets) [reply src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java]: Done

File: src/main/java/org/prebid/server/bidder/beachfront/model/BeachfrontBannerRequest.java (hunk 2 above)
Comment (RodionOrets) [reply src/main/java/org/prebid/server/bidder/beachfront/model/BeachfrontBannerRequest.java]: Changed to `Boolean`

**Review by snahornyi — APPROVED** [review]

**Pull request merged**